### ISSUE TITLE
StyleSheet Path Issue With 4.1

### ISSUE THREAD

**Issue by gordonbanderson** [wilr/silverstripe-googlesitemaps#143]
hi,

It appears this module can be installed in one of two ways

1) `composer require wilr/silverstripe-googlesitemaps "dev-master"`, this results in a stylesheet reference of `localhost/resources/wilr/silverstripe-googlesitemaps/css/style.css`, which is not found.

2) `composer require silverstripe/googlesitemaps "dev-master"`, this results in a stylesheet reference of `http://localhost/resources/wilr/silverstripe-googlesitemaps/css/style.css` when viewing sitemap.xml, which is also invalid.

I would submit a pull request but I'm not sure which path to use :)

Lastly there is a missing use statement

```
[Emergency] Uncaught Error: Class 'Wilr\GoogleSitemaps\Extensions\ArrayList' not found
GET /sitemap.xml/sitemap/SilverStripe-CMS-Model-SiteTree/1

Line 145 in /var/www/vendor/wilr/silverstripe-googlesitemaps/src/Extensions/GoogleSitemapSiteTreeExtension.php
```

Cheers,

Gordon

**Comment by wilr** [comment]
Fixed in adb3ed9

**Comment by gordonbanderson** [comment]
Confirmed, though I am not seeing stylesheet rendering (Firefox).  The XML works and that's all that matters.  Thanks for the prompt response.
